src/server/game/Entities/Object/Object.h

```diff
@@ -251,12 +251,12 @@ class FlaggedValuesArray32
 
         T_FLAGS  GetFlags() const { return m_flags; }
         bool     HasFlag(FLAG_TYPE flag) const { return m_flags & (1 << flag); }
-        void     AddFlag(FLAG_TYPE flag) { m_flags |= (1 << flag); }
-        void     DelFlag(FLAG_TYPE flag) { m_flags &= ~(1 << flag); }
+        void AddFlag(FLAG_TYPE flag) { m_flags |= (1 << flag); }
```

Comment:
Could you fix the alignment of the whole FlaggedValuesArray32 class while at it ?